Homework1/Homework1/Program.cs

```diff
@@ -0,0 +1,449 @@
+﻿namespace Homework1
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            ManagementRestaurant managementRestaurant = new ManagementRestaurant();
+
+            while (true)
+            {
+                Console.WriteLine("Choose what you want (1 - add, 2 - watch, 3 - create order, 4 - stop app)");
+                string? answerUserMethods = Console.ReadLine();
+            
+                switch (answerUserMethods)
+                {
+                    case "1":
+                        Console.WriteLine("Choose what you want (1 - add dish, 2 - add ingredients, 3 - add employees, 4 - add table)");
+                        string? answerUserAdd = Console.ReadLine();
+
+                        switch (answerUserAdd)
+                        {
+                            case "1":
+                                Console.WriteLine("Enter name of dish");
+                                string? dishName = Console.ReadLine();
+                                
+                                Console.WriteLine("Enter cost of dish");
+                                int dishCost = Int32.Parse(Console.ReadLine() ?? string.Empty);
```

Comment:
Console.ReadLine returns enpty string if user entered no symbols. Why did you use ?? statement?

Reply:
Reader says fix it like this